web/pgadmin/static/js/SecurityPages/BasePage.jsx

```diff
@@ -79,18 +83,18 @@ export default function BasePage({pageImage, title,  children, messages}) {
     });
   }, [messages]);
   return (
-    <Box className={classes.root}>
+    <StyledBox className='BasePage-root'>
```

Comment:
no need to have class on root if it is not conditional

Reply:
Done.